tests/test_home_page.py

```diff
@@ -1,64 +1,52 @@
 """Test home page."""
 
-import multiprocessing
-
 import pytest
 from selenium import webdriver
 from selenium.webdriver.remote.webelement import WebElement
-
-from dashboard import main
-
-from . import helper_test_functions as helper
-
-HOST = "127.0.0.1"
-PORT = 8000
-URL = f"http://{HOST}:{str(PORT)}"
+from tests import helper_test_functions as helper
+from tests import settings
 
 CREATE_DASHBOARD_BUTTON_ID = "create-dashboard"
 LATEST_OPENED_DASHBOARDS_ID = "latest-opened-dashboards"
 SHARED_DASHBOARDS_ID = "shared-dashboards"
 
+driver_type = (
+    webdriver.Chrome
+    | webdriver.Firefox
+    | webdriver.Safari
+    | webdriver.Edge
+    | webdriver.ChromiumEdge
+)
```

Comment:
This type alias should be used in more places, and should be probably in one of the helper modules.

Type aliases should follow the `PascalCase` naming convention.